Description: Respect extrnal CFLAGS and CPPFLAGS
Author: Alexander Inyukhin <shurick@sectorb.msk.ru>
Forwarded: yes
Last-Update: 2012-06-09
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 PREFIX?=/usr/local
 GLIBCFLAGS=-D_XOPEN_SOURCE=500 -D__STRICT_ANSI__
-CFLAGS=-pedantic -ansi -Wall -g -O0
+CPPFLAGS+=$(GLIBCFLAGS)
+CFLAGS?=-pedantic -ansi -Wall -g -O0
 OBJECTS=main.o \
 	server.o \
 	server_start.o \
@@ -32,7 +33,7 @@
 
 
 .c.o:
-	$(CC) $(CFLAGS) $(GLIBCFLAGS) -c $<
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c $<
 
 # Dependencies
 main.o: main.c main.h
